src/Common/Core/Model.php

```diff
@@ -153,49 +152,26 @@ public static function generatePassword(
      *      128px, the width will be calculated based on the aspect ratio.
      *
      * @param string $path The path wherein the thumbnail-folders will be stored.
-     * @param string $sourceFile The location of the source file.
+     * @param string $sourceFile The location of the source file
+     *
+     * @deprecated Please use the service `forkcms.utility.thumbnails` instead.
      */
     public static function generateThumbnails(string $path, string $sourceFile): void
     {
-        // get folder listing
-        $folders = self::getThumbnailFolders($path);
-        $filename = basename($sourceFile);
-
-        // loop folders
-        foreach ($folders as $folder) {
-            // generate the thumbnail
-            $thumbnail = new \SpoonThumbnail($sourceFile, $folder['width'], $folder['height']);
-            $thumbnail->setAllowEnlargement(true);
-
-            // if the width & height are specified we should ignore the aspect ratio
-            if ($folder['width'] !== null && $folder['height'] !== null) {
-                $thumbnail->setForceOriginalAspectRatio(false);
-            }
-            $thumbnail->parseToFile($folder['path'] . '/' . $filename);
-        }
+        self::get(Thumbnails::class)->generate($path, $sourceFile);
     }
 
     /**
      * Delete thumbnails based on the folders in the path
      *
      * @param string $path The path wherein the thumbnail-folders exist.
      * @param string|null $thumbnail The filename to be deleted.
+     *
+     * @deprecated Please use the service `forkcms.utility.thumbnails` instead.
```

Comment:
you removed the alias from the service definition. You need to update it here or add that alias again

Reply:
Fixed by 8fd859c